config/detekt/detekt.yml

```diff
@@ -12,3 +12,7 @@ formatting:
     active: true
     autoCorrect: true
     useTrailingCommaOnDeclarationSite: true
+style:
+  UnusedPrivateMember:
+    active: true
+    ignoreAnnotated: [ 'Preview' ]
```

Comment:
This is so it doesn't warn us about compose preview functions being unused.

Reply:
Cool!